docker-hub/creating-companies.md

```diff
@@ -0,0 +1,305 @@
+---
+description: manage companies
+keywords: company, multiple organizations, manage companies
+title: Manage a company
+---
+
+> **Note**
+>
+> Manage a company is currently in [Early Access](../release-lifecycle.md/#early-access-ea).
+> The feature is enabled for specific user groups as part of an incremental roll-out strategy.
+
+To simplify the management of Docker organizations and settings, Docker has introduced a new view that provides a single point of visibility across multiple organizations called a Company. A company can become a parent to nested child organizations. A company lets Docker Business subscribers manage their organizations and configure settings centrally. A new company owner role has also been created to control access to the company and company settings. These settings can affect all the organizations nested under the company. Up to ten unique users can be assigned a company owner role without occupying a purchased seat.
+
+Docker will work with your current Docker Organization Owners to create the Company, associate your Docker Business Organizations, and identify your Company Owner(s). Once created, users with a Company Owner role can navigate to a new page that displays the Company name, Organizations associated with the company, a list of Company Owners, and settings that include your Domain verification, Single Sign-on (SSO) connection to your identity provider, System for Cross-domain Identity Management (SCIM) setup.
+
+
+ ![company-process](images/company-process.png){: width="700px" }
+
+When a company owner makes adjustments to user management settings at the company level, this will affect all organizations associated with the company.
+
+The company owner can:
+
+- View all nested organizations.
+- Configure SSO and SCIM for all nested organizations, including SCIM Group mapping.
+- Enforce SSO log-in for all users in the company.
+- Verify a domain separately from the organization namespace.
+- Add and remove up to 10 company owners.
+- Company owners don't take a seat.
+
+A company owner role is only available if your organization has a Docker Business subscription. If you don't have a Docker Business subscription, you must first [upgrade your subscription](../subscription/upgrade.md).
+
+## Get started
+
+You’ll need to send the following information to Docker to set up your company and send it to your CSM Docker team member:
+
+- The name of your company. For example, Docker uses the company name **dockerinc**.
+- The organizations want to associate with the new company.
```

Comment:
Should this be "The organizations that you want to associate with the new company."?

Reply:
Updated